invenio_search/api.py

```diff
@@ -65,8 +66,13 @@ def search(self, user_info=None, collection=None, **kwargs):
 
         for walker in search_walkers():
             query = query.accept(walker)
-        return Results(query)
 
+        index = cfg["SEARCH_ELASTIC_COLLECTION_INDEX_MAPPING"].get(
```

Comment:
So for the INSPIRE use case as they are quite different document types, we would have one 'index' with each own mapping for Papers, Jobs, Conferences,.... Then inside of each of them there can be collections.

@tiborsimko it is not a priority at the moment to search across indices. It would be nice at some point to do something Google-like:

![2015-08-19](https://cloud.githubusercontent.com/assets/580727/9358473/98f0ddb6-468c-11e5-8507-55539efd4576.png)

But for the moment we could also achieve that by doing 2 separate searches.